### PR TITLE
task-2-components

### PR DESCRIPTION
**Pull request by gogumaUno**

### CODE REVIEW

File: src/app/modules/shared/components/search-panel/search-panel.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div>
-  <input type="text">
-  <button>Search</button>
-</div>
+<form class="search-form">
```
Comment (igor0413) [inline src/app/modules/shared/components/search-panel/search-panel.component.html]: Form based on one input. I think it's overhead.
https://www.uxbooth.com/articles/best-practices-for-search/ about design search field

File: src/app/modules/shared/components/search-panel/search-panel.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+  <input #input class="search-form__input" type="text" [placeholder]="placeholder" [value]="inputValue | async" (input)="setInputValue($event)">
```
Comment (igor0413) [inline src/app/modules/shared/components/search-panel/search-panel.component.html]: Do line wrapping!
<input #input
         class="search-form__input"
         type="text" 
         [placeholder]="placeholder" 
         [value]="inputValue | async" 
         (input)="setInputValue($event)">

Comment (igor0413) [reply src/app/modules/shared/components/search-panel/search-panel.component.html]: And I have expected to see banana in the box 
Bind [()] input field with component property

Comment (igor0413) [reply src/app/modules/shared/components/search-panel/search-panel.component.html]: But you good, You have done it through rxjs

File: src/app/components/breadcrumps/breadcrumps.component.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+@import "../../../styles/variables/color.scss";
+
+.breadcrumps-list {
+
+  list-style: none;
+  padding: 0;
+  margin: 0 0 0 1.5rem;
+  padding: 10px 0;
```
[the remark this answers is not in the document]
Comment (gogumaUno) [reply src/app/components/breadcrumps/breadcrumps.component.scss]: rem and em were removed in 5be0a87f17fb5574fe4ead402bb2c2e7cf15ebab

File: src/app/modules/shared/components/search-panel/search-panel.component.html (hunk 1 above)
Comment (gogumaUno) [reply src/app/modules/shared/components/search-panel/search-panel.component.html]: tag form was replaced with div, search button moved from the input field in 5be0a87

File: src/app/modules/shared/components/search-panel/search-panel.component.html (hunk 2 above)
Comment (gogumaUno) [reply src/app/modules/shared/components/search-panel/search-panel.component.html]: line wrapping added in 5be0a87

**Pull request merged**